rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -32,6 +33,25 @@
 import java.net.URI;
 
 public class NarayanaLRARecovery implements LRARecoveryService {
+    private static final Logger log = Logger.getLogger(NarayanaLRARecovery.class);
+
+    /**
+     * A bit of hacking to change the internals of annotations defined in LRA TCK.
+     * There is need to adjust timeout defined on the annotation definition.
+     */
+    static {
+        String[] resourceClassNames = new String[]{
+                "org.eclipse.microprofile.lra.tck.participant.api.LraResource",
+                "org.eclipse.microprofile.lra.tck.participant.api.RecoveryResource"};
```

Comment:
wouldn't be better to use getName() here?

Reply:
not using the `getName()` was intentional. If the extension is used for deployments where these resources are not provided then using of the `Class.getName()` makes the whole `NarayanaLRARecovery`failing as it tries to link a class which is not on classpath.
It's the case e.g. for `lra-test-basic` where those classes are not available.

Reply:
wouldn't it be better then to add LRA TCK dependency to the basic module then to keep track of possible renames in the dependent project? :)

Reply:
I was investigating here a bit about what's needed to do and the TCK dependency would need to be added to any module which is to use the arquillian extension. I don't want to add such requirements on users.
This is an enhancement for the TCK behaviour (and I agree it's a bit hacky but without definition of long pause time into TCK our AMS CI is currently in troubles to get running properly the LRA tests) and not any other tests.
When there is a failure on loading class (ie. the TCK tests is renamed) then if the test starts to fail it will be needed to investigate the reasons.
By me this loosely coupling of this Arq extenstion to TCK tests is desired here.